src/routes.py

```diff
@@ -299,14 +303,24 @@ async def post_operation(
         entrypoint_name = operation["parameters"]["entrypoint"]
 
         try:
-            entrypoint = crud.get_entrypoint(db, str(contract.address), entrypoint_name)
+            entrypoint = crud.get_entrypoint(
+                db, str(contract.address), entrypoint_name)
             if not entrypoint.is_enabled:
                 raise EntrypointDisabled()
 
+            # Retrieving the user_id
```

Comment:
In this context, it's not clear who you call "the user". According to what you write just after, it seems to be the user who added the contract; however, you're also testing if the user is "new" (for some definition of "new", but whatever), which, I suspect, is your way of translating the real-world example I gave you. In this example, I was giving you the idea that we needed to change the "max calls per user" condition so that it can be used to limit the activity of some users without impacting newer users, but your translation of this requirement seems rather fanciful.

Reply:
The general idea is that, in the context of this issue/PR, a user should not be considered "new" because of the age of their account, but rather because of their recent activity relatively to the sponsored contract.

However, when we talk about "user" in this context, we're definitely talking about end-users, or sponsees (users whose transactions are paid for by the API) and not contracts' owners.